Makefile

```diff
@@ -57,11 +57,13 @@ TEST_ACCEPTANCE_TAGS_ARG ?= --tags="~@disabled" --tags="~@examples"
 endif
 
 # Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifneq (,$(shell go version || echo ""))
 ifeq (,$(shell go env GOBIN))
 GOBIN=$(shell go env GOPATH)/bin
 else
 GOBIN=$(shell go env GOBIN)
 endif
+endif
```

Comment:
I am not seeing that we use `GOBIN` anywhere at all in the project - can we simply remove this block?

Reply:
Good point - I don't see it used anywhere, too... let me remove it. Thx @pedjak